docs/conf.py

```diff
@@ -186,26 +186,6 @@
               [author], 1)]
 
 
-# -- Options for the edit_on_github extension ----------------------------------------
-
-extensions += ['sphinx_astropy.ext.edit_on_github']
-
-# Don't import the module as "version" or it will override the
-# "version" configuration parameter
-from astropy import version as versionmod
-edit_on_github_project = "astropy/astropy"
-if versionmod.release:
-    edit_on_github_branch = "v{0}.{1}.x".format(
-        versionmod.major, versionmod.minor)
-else:
-    edit_on_github_branch = "master"
-edit_on_github_source_root = ""
-edit_on_github_doc_root = "docs"
-
-edit_on_github_skip_regex = '_.*|api/.*'
-
-github_issues_url = 'https://github.com/astropy/astropy/issues/'
```

Comment:
`github_issues_url` should not be deleted.

Reply:
hmm, is this a bug in sphinx-astropy to rely on this variable being set? Anyway, I'm adding it back.

Reply:
@astrofrog , can you comment on this please?